nikola/plugin_categories.py

```diff
@@ -471,3 +472,260 @@ def import_file(self):
     def save_post(self):
         """Save a post to disk."""
         raise NotImplementedError()
+
+
+class Taxonomy(BasePlugin):
+    """Taxonomy for posts.
+
+    A taxonomy plugin allows to classify posts (see #2107) by
+    classification strings. Classification plugins must adjust
+    a set of options to determine certain aspects.
+
+    The following options are class attributes with their default
+    values:
+
+    classification_name = "taxonomy":
+        The classification name to be used for path handlers.
+
+    metadata_name = "taxonomy":
+        The classification name to be used when storing the classification
+        in the metadata. If set to None, the classification won't be stored
+        in the metadata.
+
+    overview_page_variable_name = "taxonomy":
+        Variable for the overview page template which contains the list of
```

Comment:
It's the name of the variable where the list of classifications will be put into. I'll try to find a better formulation, I myself didn't even remember what I wanted to say with the current one ;)